### ISSUE TITLE
Fast pass fails for Dropdown Control: Required ARIA child role not present: textbox

### ISSUE THREAD

**Issue by RashmiBal** [microsoft/fluentui#17838]
### Environment Information

- **Package version(s)**:  "office-ui-fabric-react": "7.166.1" 
- **Browser and OS versions**: Chrome Win10

### Describe the issue:
Ran Accessibility Insights tool on my UI with Dropdown control & got this error locally. 
We are using "office-ui-fabric-react": "7.166.1" version.

#### Actual behavior:
**Fast Pass Error Details:**
Path
#Dropdown18
Snippet
`<div data-is-focusable="true" id="Dropdown18" tabindex="0" role="combobox" aria-haspopup="listbox" aria-expanded="false" aria-label="Modified On From" aria-disabled="false" class="ms-Dropdown dropdown-145">`
How to fix
Fix the following:
Required ARIA child role not present: textbox

If I edit HTML in Dev Tools to add _**role=textbox and title="some title"**_ to this element
`<span id="Dropdown18-option" class="ms-Dropdown-title title-146" aria-live="polite" aria-atomic="true">Oldest</span>`
 it fixes the issue.

**DOM created in my application has Error**
(For some reason the HTML I pasted as code isnt showing up properly here please see screenshot in the comment)
`<div class="ms-Dropdown-container root-144">
	<label>foo</label>
	<div data-is-focusable="true" id="Dropdown18" tabindex="0" role="combobox" aria-haspopup="listbox" aria-expanded="false" aria-label="Modified On From" aria-disabled="false" class="ms-Dropdown dropdown-145">
		<span id="Dropdown18-option" class="ms-Dropdown-title title-146" aria-live="polite" aria-atomic="true">Oldest</span>
		<span class="ms-Dropdown-caretDownWrapper caretDownWrapper-147">
			<i data-icon-name="ChevronDown" aria-hidden="true" class="ms-Dropdown-caretDown caretDown-161"></i>
		</span>
	</div>
</div>`

#### Expected behavior:
But this error does not repro in your official example that I see on https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown here is the codepen: https://codepen.io/RV122/pen/WNRyyjZ

**DOM created in Official Office Fabric site example does not have Error.**
_(For some reason the HTML I pasted as code isnt showing up properly here please see screenshot in the comment)_
`<div class="ms-Dropdown-container">
	<label class="ms-Label ms-Dropdown-label root-59" id="Dropdown0-label">Basic uncontrolled example</label>
	<div data-is-focusable="true" id="Dropdown0" tabindex="0" role="combobox" aria-haspopup="listbox" aria-expanded="false" aria-labelledby="Dropdown0-label Dropdown0-option" class="ms-Dropdown dropdown-43">
		<span id="Dropdown0-option" class="ms-Dropdown-title ms-Dropdown-titleIsPlaceHolder title-44" aria-live="polite" aria-atomic="true">Select an option</span>
		<span class="ms-Dropdown-caretDownWrapper caretDownWrapper-45">
			<i data-icon-name="ChevronDown" aria-hidden="true" class="ms-Dropdown-caretDown caretDown-61"></i>
		</span>
	</div>
</div>`

**Comment by RashmiBal** [comment]
![DropdownDOM](https://user-images.githubusercontent.com/46879265/114964454-9c676d80-9e23-11eb-831f-e3a702944f28.PNG)

**Comment by RashmiBal** [comment]
![DropdownDOM2](https://user-images.githubusercontent.com/46879265/114964806-4a731780-9e24-11eb-92f0-7e3e87ea8309.PNG)

**Comment by khmakoto** [comment]
I can confirm that this is a bug and it should probably be fixed.

CC: @tomi-msft

**Comment by tomi-msft** [comment]
Hi @RashmiBal, thanks for submitting the issue. After some further research, it looks like this is a false positive  on axe-core, the ruleset under accessibility insights. They're working on a fix. See this issue on their repo for more info: https://github.com/dequelabs/axe-core/issues/2505 

I would reach out to them for more info. I will go ahead and close this now
